conf/config.json

```diff
@@ -6,6 +6,20 @@
     "kafka": {
         "hosts": "127.0.0.1:9092"
     },
+    "queuePopulator": {
+        "cronRule": "*/5 * * * * *",
+        "batchMaxRead": 10000,
+        "zookeeperPath": "/queue-populator",
```

Comment:
Should we change this path for a deployment as well, in Federation? Looking here: https://github.com/scality/Federation/pull/1158/files#diff-b106b10a40ec0b4affd4efa2884f0b9fR11.

Reply:
I think it's better to leave it as the old name `replication-populator` because deployments started and changing it would mean using a new state and restarting log replication from the offset 0 (as well as having the old state lying around in zookeeper). I think it's ok to keep the old name for Federation deployments.